bazel/external/lightstep.BUILD

```diff
@@ -0,0 +1,73 @@
+load("@//bazel/external:helpers.bzl", "cc_genrule_deps")
+
+cc_library(
+    name = "lightstep",
+    srcs = [":_prefix/lib/liblightstep_core_cxx11.a"],
+    hdrs = glob([
+        "src/c++11/lightstep/**/*.h",
+        "src/c++11/mapbox_variant/**/*.hpp",
+    ]) + [
+        ":_prefix/include/collector.pb.h",
+        ":_prefix/include/lightstep_carrier.pb.h",
+    ],
+    includes = ["src/c++11", "_prefix/include"],
+    deps = ["@protobuf_bzl//:protobuf"],
+    visibility = ["//visibility:public"],
+)
+
+cc_genrule_deps(
+    name = "protobuf_deps",
+    deps = ["@protobuf_bzl//:protobuf"],
+)
+
+genrule(
+    name = "install",
+    srcs = glob(["**"]) + [
+        ":protobuf_deps",
+        "@protobuf_bzl//:well_known_protos",
+    ],
+    cmd = """
+    configure="$${PWD}/$(location :configure)"
+    protoc="$${PWD}/$(location @protobuf_bzl//:protoc)"
+    protobuf_src="$${PWD}/external/protobuf_bzl/src/"
+    protobuf_libdir="$${PWD}/$(BINDIR)/external/protobuf_bzl/"
```

Comment:
Please use caps for shell vars like `PROTOC`, `CONFIGURE` here

Reply:
`CAPITAL_CASE` variables are reserved for the OS or the shell. I'm using the Google Shell Style conventions here, which encourage `lower_case` names for variables defined by the user: https://google.github.io/styleguide/shell.xml#Naming_Conventions

Reply:
Within Envoy, I think we have a slight variation where all variables are upper case. Where there is disagreement between Google style guide and Envoy, we tend to prefer consistency (unless you want to update this everywhere). Thanks.

Reply:
I'd be fine updating this everywhere (might as well), if you're OK with the extra PRs and general commit churn.

Reply:
Fine with me. @mattklein123?